consensus/bor/bor.go

```diff
@@ -779,6 +780,7 @@ func (c *Bor) FinalizeAndAssemble(chain consensus.ChainHeaderReader, header *typ
 				return nil, err
 			}
 		}
+		log.Info("[Mining Analysis] Commit span and state-syncs", "time", time.Since(start1))
```

Comment:
Could you add block number and hash(if present) everywhere where it's about mining?

Reply:
Also could you use one field name for elapsed time? Here it is "time" and below it's 'elapsed'.

Reply:
Sure, making relevant changes. Thanks!